apps/src/maze/results/bee.js

```diff
@@ -0,0 +1,126 @@
+import Gatherer from './gatherer';
+import mazeMsg from '../locale';
+import {
+  TestResults,
+  BeeTerminationValue as TerminationValue
+} from '../../constants.js';
+
+export default class Bee extends Gatherer {
```

Comment:
Let's export `BeeHandler` here, to match how it's used in `maze/results/utils.js`.